packages/gestalt/src/Toast.js

```diff
@@ -23,10 +27,37 @@ export default function Toast({
   thumbnail,
   thumbnailShape = 'square',
   variant = 'default',
+  _dangerouslyUseDarkGray,
 }: Props): Node {
+  const { name: colorSchemeName } = useColorScheme();
+  const isDarkMode = colorSchemeName === 'darkMode';
   const isErrorVariant = variant === 'error';
-  const containerColor = isErrorVariant ? 'red' : 'white';
-  const textColor = isErrorVariant ? 'white' : undefined;
+
+  let containerColor = _dangerouslyUseDarkGray ? 'darkGray' : 'white';
+  let textColor = _dangerouslyUseDarkGray ? 'white' : 'darkGray';
+
+  if (isErrorVariant) {
+    // Error variant does not currently support dark mode
+    containerColor = 'red';
+    textColor = 'white';
+  } else if (isDarkMode) {
```

Comment:
is DarkMode the same than default variant when `_dangerouslyUseDarkGray=false`?

```
containerColor = 'white';
    textColor = 'darkGray';
```

Reply:
Yep: colors are inverted so the toasts pop against dark backgrounds.